TwitchLib.Api.Core/ApiBase.cs

```diff
@@ -364,7 +364,7 @@ private List<AuthScopes> ValidAccessToken(string accessToken)
             try
             {
                 var resp = GetRootAsync(accessToken).GetAwaiter().GetResult();
```

Comment:
can we somehow replace that call to root kraken with an actuall /validate call?

Reply:
I agree

Reply:
We're calling id.twitch.tv now :)